routes/api.js

```diff
@@ -0,0 +1,22 @@
+const express = require("express");
+const router = express.Router();
+
+const db = require("../db/models");
+const { asyncHandler } = require("./utils");
+
+const createReview = db.Review
+router.post(
+  "/recipes/:id(\\d+)/review",
+  asyncHandler(async (req, res) => {
+
+    const userId = res.locals.user.id
+    const userName = res.locals.user.userName
+
+    const recipeId = req.params.id
+    const { revContent, rating } = req.body
+    const review = await createReview.create({ userId, recipeId, revContent, rating: rating[0] })
```

Comment:
Then please change this to `const review = await Review.create({ userId, recipeId, revContent, rating: rating[0] })`

Reply:
Also can you please tell me why you are doing `rating[0]` instead of `rating`?

Reply:
Fixed the first half!
second half: We have unicode stars within a selector on the pug form which interferes with the posting to a database. Keying into rating[0] allows us to access the rating integer without the unicode value for the star.

Reply:
Ohh I see good to know. Thanks for clarifying.